docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,229 @@
+
+<!--- These gifs will have screenshots from forking, terraform plan and apply, visit azure portal and visit app service url --->
+
+### **Step 1:** Fork Cobalt Repo
+
+Initializing a repository that you own and control is recommended. Once forked, move onto the next step.
+
+* From any page within this repository, find the forked menu and fork the repo into your own repository.
+
+    ![image](https://user-images.githubusercontent.com/10041279/66366857-6e17f080-e957-11e9-8b32-266b0d4a98fc.png)
+
+### **Step 2:** Clone Repo to Local Environment
+
+You will be deploying Azure infrastructure for your local environment so you will need to have a copy of the Cobalt project locally.
+
+* From any terminal with git, use the following git command to clone your forked repo into your local environment. You can find your git repo url at the landing page of your forked repo.
+
+    ```bash
+    git clone <insert-forked-git-repo-url> # ex. git clone https://github.com/<YourGitAccout>/cobalt.git
+    ```
+
+### **Step 3:** Setup Local Environment Variables
+
+You'll need to define a `.env` file in the root of your local project. You can use our [environment template file](https://github.com/microsoft/cobalt/blob/master/.env.template) to start. This will hold all the environment variables needed to run your Cobalt deployments locally.
+
+1. Navigate to the root directory of your project and use the following command to copy the environment template file.
+
+    ```bash
+    cp .env.template .env
+    ```
+
+    ```bash
+    $ tree infra
+    ├───.env.template
+    ├───.env ## NEW FILE GENERATED FROM THE COMMAND
+    ├───modules
+    │   └───providers
+    │       ├───azure
+    │       │   ├───...
+    │       │   └───vnet
+    │       └───common
+    └───templates
+        ├───az-hello-world
+        │   └───test
+        └───backend-state-setup
+    ```
+
+1. Provide values for the environment values in `.env` which are required to authenticate Terraform to provision resources within your subscription.
+
+    ```bash
+    # Versioning
+    GO_VERSION=1.12.5
+    TF_VERSION=0.12.2
+    BUILD_BUILDID=1
+    # Subscription Values
+    ARM_SUBSCRIPTION_ID="<az-service-principal-subscription-id>"
+    ARM_CLIENT_ID="<az-service-principal-client-id>"
+    ARM_CLIENT_SECRET="<az-service-principal-auth-secret>"
+    ARM_TENANT_ID="<az-service-principal-tenant>"
+    ARM_ACCESS_KEY="<remote-state-storage-account-primary-key>"
+    # Azure Blob Storage Values
+    TF_VAR_remote_state_account="<tf-remote-state-storage-account-name>"
+    TF_VAR_remote_state_container="<tf-remote-state-storage-container-name>"
+    TF_WARN_OUTPUT_ERRORS=1
+    TF_CLI_ARGS=
+    ```
+
+1. Execute the following commands to set up the environment variables for your bash session:
+
+    ```bash
+    # these commands setup all the environment variables needed to run this template
+    DOT_ENV=<path to your .env file>
+    export $(cat $DOT_ENV | xargs)
```

Comment:
If the file has lines that start with `#`, does this work?

Reply:
It didn't for me. Good catch!

Reply:
You may want something like this:
`export $(cat .env | grep -v '^\s*#' | xargs)`

Here is the deal w/ the grep command...
 - `^\s*` - match anything that has zero or more occurrences of a white-space character
 - `^\s*#` - match anything that has zero or more occurrences of a white-space character, followed by a `#` character -i.e., this is a comment for bash
 - `-v` - inverts the match

Reply:
I echoed each command to see this play out. Looks like this will work! Thanks!